client/src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Layout } from 'antd';
+import React, { Fragment } from 'react';
+
+import styles from './Sidebar.module.css';
+import SidebarItem from './SidebarItem';
+
+const { Header, Footer } = Layout;
+
+const bookmarks = [
+  { name: 'Strona Główna', url: '#home' },
```

Comment:
Delete '#' from urls. And it's a good idea to click through your changes to be 100% they're working before commiting ;)